src/functTest/groovy/com/bmuschko/gradle/docker/AbstractFunctionalTest.groovy

```diff
@@ -34,6 +45,20 @@ abstract class AbstractFunctionalTest extends Specification {
 
     String dockerServerUrl
 
+    @Shared
+    DockerClientConfig config = DefaultDockerClientConfig.createDefaultConfigBuilder()
+        .withDockerHost("unix:///var/run/docker.sock")
```

Comment:
@orzeh I see how I can use TestConfiguration class to get the docker config. I guess @cdancy suggestion is to avoid bringing the client dependency but I'm still not finding any reference where to see how to grab a docker-client.